Dockerfile

```diff
@@ -15,24 +15,26 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
-FROM bitnami/tomcat:7.0.94 as fineract
-
-USER root
-RUN apt-get update -qq && apt-get install -y git openjdk-8-jdk wget
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/jre
+FROM openjdk:8 AS builder
 
 RUN mkdir fineract
 COPY . fineract
 
 WORKDIR fineract
 RUN sed -i 's/localhost/fineractmysql/' ./fineract-provider/src/main/resources/sql/migrations/list_db/V1__mifos-platform-shared-tenants.sql
 RUN ./gradlew clean war
-RUN mv build/libs/fineract-provider.war /opt/bitnami/tomcat/webapps
+
+
+FROM bitnami/tomcat:7.0.94 as fineract
+
+USER root
+RUN apt-get update -qq && apt-get install -y git openjdk-8-jdk wget
```

Comment:
Why are you installing git? Are you using it somewhere?

Reply:
My bad...did not see the second commit..even as you highlighted in your message.